src/transformers/modeling_utils.py

```diff
@@ -17,7 +17,7 @@
 
 import logging
 import os
-from typing import Callable, Tuple
+from typing import Callable, Sequence, Tuple
```

Comment:
we usually use `typing.Iterable`, any reason to use `Sequence` here instead?

Reply:
Changed to `Iterable`